dev/license/updatelicense.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
In such case this bash script seems to be unnecessary. Let's stick to node script alone. I'm also thinking if we could make it a grunt task, but I'm not sure about it. WDYT @jacekbogdanski?

Reply:
What's more, it seems that this file has no rights to execute.

Reply:
I'm not sure about grunt task. The main idea was to use this script on every CKEditor repo with license to update. If we put this script into grunt tasks it will be glued to https://github.com/ckeditor/ckeditor-dev repository. Maybe this script should be available at https://github.com/cksource/tools with repo path requirement? It's hard to predict path to node instance on every OS, what do you think about clean js file (without hashbang) and execution by `node updatelicense.js`?

Reply:
It'd leave it here. Clean js file without bash scripts SGTM.